### ISSUE TITLE
Wrong namespace

### ISSUE THREAD

**Issue by steadicat** [ladderlife/om-css#23]
I’m not sure if this is a regression in 0.4.4. It definitely looks familiar.

``` clojure
(defcomponent inner [props children]
  (dom/div props children))

(defcomponent outer [props children]
  (dom/div props children))

(defcomponent wrapper [props children]
  [[:.inner {}]]
  (let []
    (outer
     {:class :outer}
     (inner {:class :inner} "inner"))))
```

The above generates

``` html
<div class="module_outer_outer">
  <div class="module_inner_inner">inner</div>
</div>
```

instead of 

``` html
<div class="module_wrapper_outer">
  <div class="module_wrapper_inner">inner</div>
</div>
```

Actually, since `.outer` is not defined, the above should be:

``` html
<div>
  <div class="module_wrapper_inner">inner</div>
</div>
```

**Comment by steadicat** [comment]
Actually the bug persists even if I remove the `let`. This is definitely a regression. I’m surprised it didn’t break everything.

``` clojure
(defcomponent inner [props children]
  (dom/div props children))

(defcomponent outer [props children]
  (dom/div props children))

(defcomponent wrapper [props children]
  [[:.outer {}]
   [:.inner {}]]
  (outer
   {:class :outer}
   (inner {:class :inner} "inner")))
```

``` html
<div class="module_wrapper_outer">
  <div class="module_inner_inner">inner</div>
</div>
```

**Comment by anmonteiro** [comment]
fixed 96128a472e9c52f6147450bbc8363fb5e1d5297f
